README.md

```diff
@@ -1312,231 +1312,7 @@ ios_additional_syslog_settings { "default":
-
-##### 3750
-
-##### 4507r
-
-##### 4948
-
-##### 6503
-
-The IOS operating system needs to support the new "radius server" command, we do not use "radius-server" [link](https://www.cisco.com/c/en/us/support/docs/security-vpn/remote-authentication-dial-user-service-radius/200403-AAA-Server-Priority-explained-with-new-R.html)
-
-#### ios_stp_global
-
-##### 3650
-
-##### 3750
-
-##### 2960
-
-##### 4503
-
-##### 4507
-
-##### 4948
-
-This device does not support bridge assurance [link](https://www.cisco.com/c/en/us/td/docs/switches/lan/catalyst2960/software/release/12-2_53_se/configuration/guide/2960scg/swstp.html)
-
-#### syslog_settings
-
-Does not implement `vrf` or `time_stamp_units` as described in the netdev_stdlib type definition.
-
-#### tacacs_server
-
-##### 2960
-
-##### 3750
-
-The IOS operating system uses the deprecated "tacacs_server" syntax, we cannot use 'unset' functionality for individual fields [link](https://slaptijack.com/networking/new-style-tacacs-configuration/)
-
-#### tacacs_global
-
-The IOS operating system does not support:
-
-* enable
-* retransmit_count
-
-#### ios_additional_syslog_settings
-
-##### 4948
-
-The `origin-id` command is not avalible on th above machine.
-
-### Anomalies in Cisco CLI
-
-#### ntp_server
-
-It has been noted that NTP Server configuration may allow multiple entries of the same NTP Server address with different Source Interfaces
-
-For example:
-````
-ntp server 1.2.3.4 key 42
-ntp server 1.2.3.4 key 94 source Vlan42
-ntp server 1.2.3.4 key 50 source Loopback42
-````
-While Puppet Resource will obtain all entries, Puppet Apply compares against the first entry found with the same name.
-
-##### Workaround
-
-Send an ensure 'absent' manifest to remove all ntp servers of the same name, before rebuilding the ntp server configuration:
-
-````Puppet
-    ntp_server { '1.2.3.4':
-      ensure => 'absent',
-    }
-````
-
-followed by:
-
-````Puppet
-    ntp_server { '1.2.3.4':
-      ensure => 'present',
-      key => 94,
-      prefer => true,
-      minpoll => 4,
-      maxpoll => 14,
-      source_interface => 'Vlan 42',
-    }
-````
-
-Any edits can be made by referencing the same ntp_server name and source_interface.
+The module works against a broad set of IOS based devices, but we can't test against all device types - there are simply too many. However, we do run tests against a range of physical devices from the Catalyst family - both IOS and IOS-XE - in our continuous integration pipelines. If you experience errors or missing abstractions, provide the error details (if present) via --debug --trace and a copy of the sanitized configuration (both Puppet and Cisco CLI). IOS devices vary in their capabilities depending on the hardware and software. For details on the physical devices that we test against and limitations please see the [Test Matrix](./TestMatrix.md). Note that our initial development and testing has focussed on switches, with routers and firewalls to follow-on.
```

Comment:
The link to test matrix should be full like to aid with forge users. Should be something like https://github.com/puppetlabs/cisco_ios/blob/master/TestMatrix.md

Reply:
Good call - I updated the URL to be a full one.